README.md

```diff
@@ -37,8 +37,46 @@ pnpm i18n:get-translations
 
 # Builds and serves assets with hot-reload
 pnpm dev
+
+```
+
+### Using HTTPS Locally
+
+To enable SSL support in local development, use the `pnpm dev:secure` command. This requires you to have a private key and certificate in the root of the repository with the following names:
+
+```shell
+localhost+1-key.pem # The private key file
+localhost+1.pem # The certificate file
+```
+
+The easiest way to create these files is with a local development tool called [mkcrt](https://github.com/FiloSottile/mkcert). First make sure you have [mkcert installed](https://github.com/FiloSottile/mkcert#installation) and activated with `mkcert -install`. Then use `mkcert` to create a certificate for `localhost` and for the external IP address used by Nuxt's development process. That command looks like this:
```

Comment:
Nice to know about `mkcert`. Maybe we can also use this in WordPress/openverse-api#574 to avoid the untrusted certificate issue.

Reply:
Yes, definitely! mkcert actually sets up a local certificate authority, it's a really great tool.